bwe/RateController.cpp

```diff
@@ -167,9 +167,7 @@ void RateController::onReportBlockReceived(uint32_t ssrc,
     {
         if (!_lastProbe.empty() && item.reportNtp == _lastProbe.ntp && item.type == PacketMetaData::SR)
```

Comment:
What if the RB referes to another previous SR that is now better confirmed along with the _lastProbe. 
We only search deeper if the RB refers to this lastProbe.